src/MiniCover/Commands/Options/WorkingDirectoryOption.cs

```diff
@@ -22,7 +22,7 @@ protected override DirectoryInfo GetOptionValue()
             var directory = new DirectoryInfo(workingDirectoryPath);
```

Comment:
why not using the return of Diretory.CreateDirectory(workingDirectoryPath) instead of recreating a DirectoryInfo